site/pages/theme-topic-en.hbs

```diff
@@ -52,6 +52,14 @@
 			<div class="col-md-4">
 				<h3><a href="#">[Subtopic hyperlink text]</a></h3>
 				<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
```

Comment:
Please change the templated title in a way it suggested an action. At least for the description, like: Summary. Select a sub-category in the following drop down